doc/source/coding-style/formatting-tools.rst

```diff
@@ -0,0 +1,241 @@
+Code Style Tools
+================
+
+There are plenty of tools for checking code style. This section presents some of
+the most popular ones in the Python ecosystem. A minimum configuration is
+provided for each one so you can easily include them in your PyAnsys project.
+
+Most of the tools presented can be configured using :ref:`the
+\`\`pyproject.toml\`\` file`, avoiding dotfiles and thus leading to a much
+cleaner root project directory.
+
+
+Black
+-----
+`Black`_ is the most popular code formatter in the Python community, as it is
+being maintained by the Python Software Foundation. It allows for a minimum
+configuration to ensure that Python code format looks almost the same across
+projects. 
```

Comment:
```suggestion
`Black`_ is the most popular code formatter in the Python community because it is
maintained by the Python Software Foundation. It allows for a minimum
configuration to ensure that the Python code format looks almost the same across
projects. 
```